### PR TITLE
Publish 84628, 84918

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: manuscripts.json [hunk 1]
```diff
@@ -97,6 +97,32 @@
...
       "msas": ["Neuroscience"],
       "pdfUrl": "https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/10.1101/2022.11.08.515698/2022.11.08.515698.pdf"
+    },
+    "10.1101/2022.10.28.514241": {
+      "preprintDoi": "10.1101/2022.10.28.514241",
+      "status": {
+        "articleType": "Reviewed Preprint",
+        "status": "Published from the original preprint after peer review and assessment by eLife.",
+        "timeline": [
+          { "name": "Reviewed Preprint posted", "date": "2023-01-02" },
```
Comment (FAtherden-eLife) [inline manuscripts.json:107]: I think this should be `"date": "2023-02-02"` if we're aiming to publish this today.

File: manuscripts.json [hunk 2]
```diff
@@ -97,6 +97,32 @@
...
+      },
+      "msas": ["Genetics and Genomics", "Neuroscience"]
+    },
+    "10.1101/2022.12.01.518662": {
+      "preprintDoi": "10.1101/2022.12.01.518662",
+      "status": {
+        "articleType": "Reviewed Preprint",
+        "status": "Published from the original preprint after peer review and assessment by eLife.",
+        "timeline": [
+          { "name": "Reviewed Preprint posted", "date": "2023-01-02" },
```
Comment (FAtherden-eLife) [inline manuscripts.json:120]: As above for the RP posted date

**Review by FAtherden-eLife — CHANGES_REQUESTED** [review]
I think RP posted dates should be `"date": "2023-02-02"` instead of `"date": "2023-01-02"`

**Review by FAtherden-eLife — APPROVED** [review]

File: manuscripts.json (hunk 1 above)
Comment (nlisgo) [reply manuscripts.json]: Updated!

File: manuscripts.json (hunk 2 above)
Comment (nlisgo) [reply manuscripts.json]: Updated!

**Review by scottaubrey — APPROVED** [review]

**Pull request merged**